### ISSUE TITLE
Weaponizing stewart platform so it can defend itself

### ISSUE THREAD

**Issue by rcr2f** [correll/advancedrobotics#43]

**Comment by Jorge-C** [comment]
After talking to some independent experts, we've been recommended the following self-defense platform: http://www.toysrus.com/product/index.jsp?productId=46534416&prodFindSrc=pn&cp=28760506

**Comment by ryanleonard** [comment]
I agree, but I think that if we got that, some modifications might be in order:

https://www.youtube.com/watch?v=pu_0qTyuJLY
